docs/usage/configuration-options.md

```diff
@@ -1834,11 +1834,11 @@ For example, GitHub might automerge a Renovate branch even if it's behind the ba
 
 ## platformCommit
 
-Supports only GitHub App mode and not when using Personal Access Tokens.
+Only use this option if you run Renovate as a [GitHub App](https://docs.github.com/en/developers/apps/getting-started-with-apps/about-apps).
+It does not apply when you use a Personal Access Token as credential.
 
-To avoid errors, `gitAuthor` or `gitIgnoredAuthors` should be manually adjusted accordingly.
-
-The primary reason to use this option is because commits will then be signed automatically if authenticating as an app.
+You can use `platformCommit` to automatically sign the commits that Renovate creates.
+To avoid errors, set the `gitAuthor` or `gitIgnoredAuthors` or both.
```

Comment:
From reading the text it's not clear _why_ I need to set `gitAuthor` or `gitIgnoredAuthors` to prevent errors.